### ISSUE TITLE
Unable to find library when compiling examples

### ISSUE THREAD

**Issue by monarrk** [RustAudio/rust-jack#116]
I'm attempting to compile [this example](https://github.com/RustAudio/rust-jack/blob/master/examples/show_midi.rs) but continue to get linker errors about missing libraries. I'm running OpenBSD with stable rust 1.38.0 and I do have JACK installed.

The relevant part of the error is this (if you need more I'm happy to provide):
```  = note: ld: error: unable to find library -ljack
          cc: error: linker command failed with exit code 1 (use -v to see invocation)
```

Thank you

**Comment by wmedrano** [comment]
Sorry, I'm not familiar with OpenBSD. Can you post the extended error with -v?

**Comment by wmedrano** [comment]
This should be fixed with `0.10.0` release OR it will at least provide a better error.
